astronomy/лидов_古在_test.cpp

```diff
@@ -0,0 +1,165 @@
+﻿
+#include <memory>
+
+#include "astronomy/butcher_mercury_orbiter_1119.hpp"
+#include "astronomy/orbital_elements.hpp"
+#include "astronomy/frames.hpp"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "ksp_plugin/integrators.hpp"
```

Comment:
`astronomy` should not depend on `ksp_plugin`, that's going to introduce horrible compilation dependencies.  Just replicate the parameters as needed.  (It's more robust too: if we decide to change a bit the way that we downsample, we should not break this test.)